charts/ocean-kubernetes-controller/Chart.yaml

```diff
@@ -17,6 +17,6 @@ dependencies:
 - name: metrics-server
   version: 3.11.0
   repository: https://kubernetes-sigs.github.io/metrics-server
-  condition: metrics-server.enabled
+  condition: metrics-server.deployChart
```

Comment:
??

condition: (optional) A yaml path that resolves to a boolean, used for enabling/disabling charts (e.g. subchart1.enabled )

Reply:
https://github.com/spotinst/charts/blob/main/charts/ocean-kubernetes-controller/values.yaml#L169
@oded7hoffman

Reply:
that's the right yaml path that decides whether to deploy metrics-server or not.